enrollment-server-onboarding-domain-model/src/main/java/com/wultra/app/enrollmentserver/model/enumeration/ErrorOrigin.java

```diff
@@ -34,6 +34,7 @@ public enum ErrorOrigin {
 
     PROCESS_LIMIT_CHECK,
 
-    USER_REQUEST
+    USER_REQUEST,
 
+    FINAL_VALIDATION
```

Comment:
I guess that this should be propagated to documentation because of reporting.

Reply:
Prepared change of the documentation https://github.com/wultra/powerauth-product/pull/86